modules/storages/app/components/storages/admin/sidebar_component.html.erb

```diff
@@ -2,7 +2,10 @@
   component_wrapper do
     render(Primer::OpenProject::BorderGrid.new) do |border_grid|
       border_grid.with_row { render(Storages::Admin::Sidebar::HealthStatusComponent.new(storage: @storage)) }
-      border_grid.with_row { render(Storages::Admin::Sidebar::HealthNotificationsComponent.new(storage: @storage)) }
+      
+      if @storage.automatic_management_enabled?
+        border_grid.with_row { render(Storages::Admin::Sidebar::HealthNotificationsComponent.new(storage: @storage)) }
+      end
```

Comment:
🟢 _fyi:_ the HealthNotificationsComponent was already using [conditional rendering](https://viewcomponent.org/guide/conditional_rendering.html) based on `@storage.automatic_management_enabled?` but perhaps this is more clear- my impression is that `border_grid.with_row {}` does not modify the DOM if the render result is nil _*unverified_

https://github.com/opf/openproject/blob/dev/modules/storages/app/components/storages/admin/sidebar/health_notifications_component.rb#L41-L43

Reply:
Sadly, the separator of the border grid is shown, even when `render` returns `nil`. If I encapsulate the render method like this, the separator is not shown.

Do you have an idea, how to solve this with other means?

Reply:
Ah, yeah- I'd leave it out tbh- my proposal wasn't fully verified either way. Cheers for looking into it! 👍🏾